File: app/src/main/java/network/DownloadImageHelper.java
package network;

import android.content.Context;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.example.mymovies.MainActivity;

import DAO.Variables;

public class DownloadImageHelper {
    private ImageView imageView;
    private Context context;
    private String imagePath;

    public DownloadImageHelper(ImageView imageView,  String imagePath) {
        this.imageView = imageView;
        this.imagePath = imagePath;
        setImage();
    }

    private void setImage() {
        Glide
                .with(imageView.getContext())
                .load(Variables.IMG_URL + imagePath)
                .into(imageView);
    }
}

File: app/src/main/java/com/example/mymovies/utils/DownloadImageHelper.kt
package com.example.mymovies.utils

import android.content.Context
import android.graphics.drawable.Drawable
import android.widget.ImageView
import com.bumptech.glide.Glide
import com.example.mymovies.BuildConfig
import javax.inject.Inject

class DownloadImageHelper @Inject constructor(
        private val context: Context
) {
    fun loadImage(imageView: ImageView, imagePath: String) {
        Glide.with(imageView.context)
                .load(BuildConfig.IMAGE_URL + imagePath)
                .into(imageView)
    }

    fun getDrawable(imagePath: String): Drawable =
            Glide.with(context)
                    .load(BuildConfig.IMAGE_URL + imagePath)
                    .submit()
                    .get()
}